### DISCUSSION TITLE
[Q] is possible to change the cell comment box size

### DISCUSSION THREAD

**Discussion by nandakumar-a** [exceljs/exceljs#2413] [Q&A]
## 💬 Questions and Help

i tried cell comments as per the document it is created on excel but some of my text are hide in the box because it is small. is possible to change the comment text box size on exceljs?

![Screenshot (25)](https://user-images.githubusercontent.com/40513758/93337491-d39e7d80-f846-11ea-91ab-f7654e02a44b.png)

some text are hide in the above screenshot. i am resized using edit option in ms office 365 are shown below

![Screenshot (26)](https://user-images.githubusercontent.com/40513758/93337635-05174900-f847-11ea-97a3-9bb196601d2f.png)

using exceljs: ExcelJS 14-07-2020

**Comment by SyedSalmanNizami** [comment]
I have the same issue as pointed out by NandaKumar . Is there a way to dynamically adjust the width or size of the cell comment based on the content of it ?

**Comment by chdh** [comment]
Duplicate of #981?

**Comment by eminenceit** [comment]
any update on this issue please? This is an important feature!

**Comment by nandakumar-a** [comment]
any update on this issue?

**Comment by meurling** [comment]
any update on this issue?

**Comment by sillo1991** [comment]
any update???

**Comment by pescoboza** [comment]
Any update?
